README.md

```diff
@@ -98,42 +105,27 @@ The above configuration file can cope with a development and production version
 
 ### Update the aurelia configuration file
 
-In your aurelia configuration file, add the plugin and inject the aurelia-auth security configuration file :
-```js
-import config from './authConfig';
+In your aurelia configuration file, add the plugin and inject the aurelia-auth security configuration file for your protected and unprotected endpoints. 
+
+```javascript
+import authConfig from './authConfig';
+
 export function configure(aurelia) {
   aurelia.use
-    .standardConfiguration()
-    .developmentLogging()
-    .plugin('aurelia-animator-css')
-    .plugin('spoonx/aurelia-auth', (baseConfig)=>{
-         baseConfig.configure(config);
+    /* Your other plugins and init code */
+    .plugin('spoonx/aurelia-api', configure=>{
+      configure
```

Comment:
I agree completely, I just think we should describe why these options are added like this. Link to the api docs, or perhaps explain that the name of the endpoint is used in baseConfig.configure(authConfig);